prime-router/metadata/schemas/HL7/hl7-ingest-covid-19.schema

```diff
@@ -80,3 +80,18 @@ elements:
     natFlatFileField: Patient_state
     hhsGuidanceField:
     hl7Field: PID-11-4
+
+  - name: ordering_facility_zip_code
```

Comment:
I don't think this would work correctly?

The ordering_facility_state uses the `mapper: use(ordering_facility_state, ordering_facility_state_zip_lookup)` and `ordering_facility_state_zip_lookup` isn't defined. I think that needs to be where you have `ordering_facility_zip_code` defined.

Reply:
Good catch. Some how using the paste and cotrol z did something. Thanks!